src/module.c

```diff
@@ -846,7 +846,7 @@ int64_t commandKeySpecsFlagsFromString(const char *s) {
         else if (!strcasecmp(t,"INSERT")) flags |= CMD_KEY_INSERT;
         else if (!strcasecmp(t,"UPDATE")) flags |= CMD_KEY_UPDATE;
         else if (!strcasecmp(t,"DELETE")) flags |= CMD_KEY_DELETE;
```

Comment:
Also, we should make these all the same case

Reply:
yes, i see i used upper-case letters for `RW`, `RO`, and also for `INSERT`, `UPDATE`, but not for channel and incomplete.
unlike COMMAND INFO, in which case they're all lower-case, except `RW`, `RO` which are initials for two words.
i suppose we should consolidate that (COMMAND INFO and module API should be the same), which way do you prefer?

Reply:
i now noticed this is case-insensitive compare. but still fixed it to be similar to COMMAND command, and only use capital letters for the initials